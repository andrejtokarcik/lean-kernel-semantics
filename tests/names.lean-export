1 #NS 0 foo
2 #NS 1 bla
3 #NI 2 1
4 #NS 3 boo
