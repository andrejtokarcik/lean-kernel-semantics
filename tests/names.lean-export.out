<T>
  <k>
    .
  </k>
  <names>
    0 |-> *empty*
    1 |-> foo
    2 |-> foo bla
    3 |-> foo bla 1
    4 |-> foo bla 1 boo
  </names>
  <universes>
    0 |-> 0
  </universes>
  <expressions>
    .Map
  </expressions>
  <exit-code>
    0
  </exit-code>
</T>
